components/brave_ads/browser/notification_helper_android.cc

```diff
@@ -28,6 +29,11 @@ bool NotificationHelperAndroid::IsNotificationsAvailable() const {
 // Starting in Android 8.0 (API level 26), all notifications must be
 // assigned to a channel.
 bool NotificationHelperAndroid::IsBraveAdsChannelEnabled() const {
+  char osVersion[PROP_VALUE_MAX+1];
```

Comment:
we should be using snake_case and not camelCase otherwise will fail linter checks